assets/msalv2.js

```diff
@@ -79,29 +108,12 @@ var aadOauth = (function () {
 
     // Try to sign in silently, assuming we have already signed in and have
     // a cached access token
-    const account = getAccount();
-    if (account !== null) {
-      try {
-        // Silent acquisition only works if we the access token is either
-        // within its lifetime, or the refresh token can successfully be
-        // used to refresh it. This will throw if the access token can't
-        // be acquired.
-        const silentAuthResult = await myMSALObj.acquireTokenSilent({
-          scopes: tokenRequest.scopes,
-          prompt: "none",
-          account: account,
-          extraQueryParameters: tokenRequest.extraQueryParameters
-        });
-
-        authResult = silentAuthResult;
-
-        // Skip interactive login
-        onSuccess(authResult.accessToken ?? null);
+    await silentlyAcquireToken()
 
-        return;
-      } catch (error) {
-        console.log(error.message)
-      }
+    if(authResult != null) {
+      // Skip interactive login
+      onSuccess(authResult.accessToken ?? null);
```

Comment:
Why ?? Null

isn’t that the same thing as not providing the ?? Operator completely?

Reply:
This is part of the original code that was there before. According to Javascript documentation, the ?? operator will return the right-hand value in case of `null` or `undefined`, so this code would always turn an `undefined` value into `null`.

I've had my fair share of issues with ?? in Javascript, so I prefer not to use it, but since this code was there before my changes, I left it in place.

Let me know how to proceed.

Reply:
Keep it as is.